client/GameComponents/AutoPassMenu.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+class AutoPassMenu extends React.Component {
+    onToggle(option, value) {
+        if(this.props.onToggle) {
+            this.props.onToggle(option, !value);
+        }
+    }
+
+    render() {
+        return (
+            <div className='panel auto-pass'>
+            Auto pass action windows
+                <div className='checkbox'>
+                    <label>
+                        <input type='checkbox' checked={ this.props.options.plot } onChange={ this.onToggle.bind(this, 'plot', this.props.options.plot) } />Plots Revealed
```

Comment:
This is nitpicky, but since there are so many of these and they're all the same, it may be more readable to express this as an array and then use a loop to render them. e.g.
```javascript
let actionWindows = [
  { flag: 'plot', text: 'Plot Revealed' },
  { flag: 'draw', text: 'Draw Phase' },
  // ...
];

_.map(actionWindows, actionWindow => (
  <div className='checkbox'>
    <label>
      <input type='checkbox' checked={ this.props.options[actionWindow.flag] } ... />{actionWindow.text}
    </label>
  </div>
));
```